pkg/media/opus/opus.go

```diff
@@ -65,11 +66,11 @@ func (d *decoder) SampleRate() int {
 func (d *decoder) WriteSample(in Sample) error {
 	n, err := d.dec.Decode(in, d.buf)
 	if err != nil {
-		// Some workflows (concatenating opus files) can cause a suprious decoding error, so ignore single corruption errors
-		if !strings.Contains(err.Error(), "corrupted stream") || d.successiveErrorCount >= 1 {
+		// Some workflows (concatenating opus files) can cause a suprious decoding error, so ignore small amount of corruption errors
+		if !errors.Is(err, opus.ErrInvalidPacket) || d.successiveErrorCount >= 5 {
 			return err
 		}
-		log.Debugw("opus decoder failed decoding a sample")
+		logger.Debugw("opus decoder failed decoding a sample")
```

Comment:
Hmm, we should probably start passing logger to `Decode` and `Encode` constructors then. Otherwise will be hard to track down the call where this happened. I can do it in a separate PR, if you want.